src/containers/FooterContainer.vue

```diff
@@ -1,12 +1,62 @@
 <template>
-    <footer>
-    </footer>
+  <footer>
+    <Dialog
+      header="Header"
+      footer="Footer"
+      v-model:visible="display"
+      :style="{ width: '50vw', height: '65vh', justifyContent: 'space-between' }"
+    >
+      <template #header>
+        <h3>Report a bug</h3>
+      </template>
```

Comment:
Either remove this template or remove the `header` prop, since they set the same thing

Reply:
Should also remove the `footer` prop